cgmes/cgmes-model/src/main/resources/CIM16.sparql

```diff
@@ -264,7 +264,8 @@ SELECT *
 # query: connectivityNodeContainers
 SELECT *
 WHERE {
-{ GRAPH ?graphEQ {
+# FIXME(Luma) when we fix voltage levels VL definitions may be in a different graph from the original Bay
+# { GRAPH ?graphEQ {
```

Comment:
Do we want to modify this file or introduce a new file with less strict requirements?

Reply:
In this case we could directly modify the file, as we are just iterating over all connectivity node containers in all graphs.

We generally preserve graph clauses to limit the search of related data and improve performance. Here we were only implicitly enforcing that bays must be in the same graph of its voltage level, and removing the graph clause should not have a big impact. To be measured, anyway.

Reply:
Ok to remove the GRAPH clause, and we will keep a comment that refers that the attributes queried in this clause are found in the EQ instance files.